docs/platforms/javascript/common/session-replay/issue-types.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Replay Issues
+sidebar_order: 4300
+notSupported:
+  - javascript.cordova
+  - javascript.wasm
+  - javascript.bun
+  - javascript.deno
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+description: "Learn about the Issue types that Session Replay can detect."
+---
+
+A _[replay issue](/product/issues/issue-details/replay-issues/)_ is an issue detected using captured Session Replay data. If your application is configured with [Session Replay](/product/session-replay/), problems will be detected on the server side during replay ingest, and grouped into issues. We group similar events into issues based on a fingerprint. For replay issues, a fingerprint is primarily based on the problem type and the url or transaction name where the problem occurred.
+
+You can configure which issue types are created by visiting Project Settings > Replay and toggling detection on or off for each issue type.
+
+![Replay settings page](./img/replay-settings.png)
+
+## Configuring Rage Click Issues
```

Comment:
theres a min SDK version to create rage click issues. this is important for set up. lets mention it here?